src/components/activitylog.js

```diff
@@ -61,20 +61,13 @@ function renderList(elem, apiClient, result) {
 }
 
 function reloadData(instance, elem, apiClient, startIndex, limit) {
-    if (startIndex == null) {
-        startIndex = parseInt(elem.getAttribute('data-activitystartindex') || '0', 10);
-    }
-
-    limit = limit || parseInt(elem.getAttribute('data-activitylimit') || '7', 10);
+    startIndex ??= parseInt(elem.getAttribute('data-activitystartindex') || '0', 10);
+    limit ||= parseInt(elem.getAttribute('data-activitylimit') || '7', 10);
```

Comment:
the `??=` and `||=` operators probably produce some gnarly code when babelified which is something we've been trying to avoid lately

Reply:
`x ||= y` is probably fine : `x || (x = y)`

`x ??= y` produces `var _x; (_x = x) !== void 0 && _x !== null ? _x : (x = y)`. So I'd only use it if `||` doesn't fit. And probably if `y` is harder to calculate than checking 2 conditions.

Reply:
Converted it to use ||=